### ISSUE TITLE
Missing empty line after control statement

### ISSUE THREAD

**Issue by czjvic** [squizlabs/PHP_CodeSniffer#3222]
Missing check for empty line after control structure.

**Code sample - no phpcs finding (but should be)**
```php
if ($foo) {
    echo "bar";
}
$foo = "bar"
```

**Code sample - expected result after phpcbf**
```php
if ($foo) {
    echo "bar";
}
                                        <<<< this line must be completely empty
$foo = "bar"
```

**To reproduce**
1. Create a file called `test.php` with the "Code sample - no phpcs finding (but should be)" above...
2. Run `phpcs test.php`
3. No error message displayed - expected message "Missing empty line after control statement."

**Additional context**
I am 100% sure, that there was a rule, which covers this behavior, but this functionality was lost several years ago.

**Comment by gsherwood** [comment]
The sniff is still there; it's `Squiz.WhiteSpace.ControlStructureSpacing`

But unless you are running with the Squiz standard or a custom coding standard that uses it, you won't be getting it by default.

Try running `phpcs test.php -v` and see what standard you are using.

**Comment by czjvic** [comment]
This is my ruleset.
```xml
<?xml version="1.0" encoding="UTF-8"?>
<ruleset name="Test">
    <arg value="ps"/>
    <rule ref="PSR2"/>
    <rule ref="Squiz.WhiteSpace.ControlStructureSpacing" />
</ruleset>
```
With this ruleset the report is empty.

It looks that following lines in the PSR2.xml affects my ruleset (disable required feature):
```xml
    <rule ref="Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen"/>
    <rule ref="Squiz.WhiteSpace.ControlStructureSpacing.SpacingBeforeClose"/>
```

**Comment by jrfnl** [comment]
@czjvic I haven't tested, but could you test with the below - I believe that should overrule the selective enabling of the error codes done in PSR2:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<ruleset name="Test">
    <arg value="ps"/>
    <rule ref="PSR2"/>
    <rule ref="Squiz.WhiteSpace.ControlStructureSpacing">
        <severity>5</severity>
    </rule>
</ruleset>
```

**Comment by czjvic** [comment]
@jrfnl Thank you very much. It works.
